docs/recipes/recipe-marketing-attribution/index.md

```diff
@@ -225,25 +193,132 @@ FROM derived.marketing_attribution
 GROUP BY 1;
 ```
 
-## Let's break down what you've done
+## 2. Track events to create a real dataset
+
+When you've completed this section, you should learn something about how you acquire real customers!
+
+#### Auto-track page views via GTM
+[Tracking real page view events](https://docs.snowplow.io/docs/try-snowplow/tracking-events-with-try-snowplow/) is very simple, especially if you have Google Tag Manager connected to your website (or your personal blog if you just want to see how it works!) - just copy the snippet from the Snowplow UI into a custom HTML tag.
+
+![](images/gtm.gif)
+
+#### Run the same SQL on this dataset
+Now run the same SQL snippets from step 1 on this new data in Postgres and don't forget to update the conversion logic to something that makes sense for your website such as:
+
+```sql
+CREATE TABLE derived.marketing_attribution AS(
+    WITH session_aggregations AS (
+
+        SELECT
+        ...
+            SUM(CASE WHEN ev.page_urlpath = '/thank-you/' THEN 1 ELSE 0 END) AS conversions
+
+        ...
+```
 
-- With the out of the box tracking, you have automatically captured the marketing and referrer information for each page view and session.
-- By instrumenting a custom conversion event, you are collecting all the data you need to get started with attribution.
-- By running a simple query that attributes the conversions to the marketing channels based on three popular attribution models, you can explore how Snowplow enables you to take control of your marketing attribution.
+#### [Optional] For more confidence, track a dedicated conversion event
 
-## What you might want to do next
+<details>
+  <summary>
+Design and implement a `conversion` event
+  </summary>
```

Comment:
Backticks don't work in `summary` tags, you'll need to swap to a `<code>` tag around it

Reply:
Updated